Lib/fontbakery/profiles/universal.py

```diff
@@ -330,22 +330,37 @@ def is_up_to_date(installed, latest):
     """,
     proposal = 'https://github.com/googlefonts/fontbakery/issues/2093'
 )
-def com_google_fonts_check_fontbakery_version():
+def com_google_fonts_check_fontbakery_version(font):
     """Do we have the latest version of FontBakery installed?"""
-    import json
     import requests
     import pip_api
 
-    pypi_data = requests.get('https://pypi.org/pypi/fontbakery/json')
-    latest = json.loads(pypi_data.content)["info"]["version"]
+    try:
+        response = requests.get('https://pypi.org/pypi/fontbakery/json')
+
+    except requests.exceptions.ConnectionError as err:
+        return SKIP, Message(
+            "connection-error",
+            f"Request to PyPI.org failed with this message:\n{err}",
+        )
+
+    status_code = response.status_code
+    if status_code != 200:
+        return SKIP, Message(
```

Comment:
same here.

Reply:
I'm OK with changing this to FAIL, even though there's nothing the user will be able to do to fix the problem.
In this situation the user has a working internet connection, but the response from PyPI.org was not successful.

Reply:
Changed to FAIL in fb528be90d99aeaf8bf9b4c50c8ef3e1c2504c2a